main.go

```diff
@@ -44,9 +47,12 @@ func (c *Criu) Prepare() error {
 	srv := os.NewFile(uintptr(fds[1]), "criu-xprt-srv")
 	defer srv.Close()
 
+	out := new(bytes.Buffer)
 	args := []string{"swrk", strconv.Itoa(fds[1])}
 	// #nosec G204
 	cmd := exec.Command(c.swrkPath, args...)
+	cmd.Stdout = out
+	cmd.Stderr = out
```

Comment:
Would it make more sense to capture stdout and stderr separately? I'd assume you wouldn't want random info logs to clutter the error logs while debugging

Reply:
@snprajwal I believe that CRIU uses only one file descriptor for logging ([`log_get_fd()`](https://github.com/checkpoint-restore/criu/blob/bf8c134714d6e041261b5129561eeb49aa734ec1/criu/log.c#L79)).

Reply:
yeah I wasn't sure to be honest, I did a quick look in the criu source and found perror calls there that I might be interested i. But I wasn't sure about stdout, if it doesn't log there it should not make a difference but I could also drop it back to /dev/null like it is right now.